### ISSUE TITLE
Test coverage for ENCORE needs to be increased

### ISSUE THREAD

**Issue by jbarnoud** [MDAnalysis/mdaencore#29]
## Expected behavior ##

Analysis code base is covered with test, including ENCORE.

## Actual behavior ##

Some of ENCORE's untested code appear to be dead code. See https://github.com/MDAnalysis/mdanalysis/pull/3479#issuecomment-1029249778

Ping @mtiberti and @wouterboomsma

**Comment by wouterboomsma** [comment]
@jbarnoud, I'm trying to get an overview over outstanding issues in Encore (we're looking into finding a new maintainer for the code). Do you know if this current issue is still relevant or was it solved by https://github.com/MDAnalysis/mdanalysis/pull/3621?

**Comment by IAlibay** [comment]
@wouterboomsma would you be happy to have a call with the MDA coredevs to discuss this further at some point? We would definitely love to have someone come here and help maintain encore.

We can probably also explain the current move towards mdaencore and how it falls without our commitments to develop an ecosystem of downstream tools

**Comment by wouterboomsma** [comment]
Yes. That would be great, thanks. We'll contact all the original authors of the ENCORE paper for a meeting. If possible, it would be great if one or more of the MDA coredevs could briefly join in and explain the vision for the downstream tool ecosystem. Shall I reach out through email?

**Comment by IAlibay** [comment]
> Yes. That would be great, thanks. We'll contact all the original authors of the ENCORE paper for a meeting. If possible, it would be great if one or more of the MDA coredevs could briefly join in and explain the vision for the downstream tool ecosystem. Shall I reach out through email?

If you can reach out to mdanalysis@numfocus.org it will go out to all the coredevs so we can make sure at least one of us can join in.
